contrib/debian/netdata.postrm

```diff
@@ -16,45 +16,31 @@ dpkg-maintscript-helper dir_to_symlink \
 	/var/lib/netdata/www/static /usr/share/netdata/www/static 1.18.1~ netdata -- "$@"
 
 case "$1" in
-	remove)
-		;;
+remove) ;;
 
+\
```

Comment:
This confuses me. Why is it needed?
```suggestion
```

Reply:
Oh. actually the auto-formatter is doing this shfmt Out-of-my-control unless I go to the effort of tweaking the default settings for shfmt or something. I haven't looked into it tbh.

Reply:
Oh. actually the auto-formatter is doing this shfmt Out-of-my-control unless I go to the effort of tweaking the default settings for shfmt or something. I haven't looked into it tbh.